app/components/search-facet-daterange/component.js

```diff
@@ -38,6 +37,7 @@ export default Ember.Component.extend({
                 this.send('clear');
             });
         });
+        this.filterUpdated();
```

Comment:
This triggers an unhappy warning:
> DEPRECATION: A property of <ember-share@view:-outlet::ember421> was modified inside the didInsertElement hook. You should never change properties on components, services or models during didInsertElement because it causes significant performance degradation.

Could you initialize `pickerValue` in `init` instead?